pyspectral/rayleigh.py

```diff
@@ -344,46 +304,66 @@ def reduce_rayleigh_highzenith(zenith, rayref, thresh_zen, maxzen, strength):
         be linearly scaled, from one at `thresh_zen` to zero at `maxzen`.
         """
         LOG.info("Reducing Rayleigh effect at high zenith angles.")
-        factor = 1. - strength * where(zenith < thresh_zen, 0, (zenith - thresh_zen) / (maxzen - thresh_zen))
+        factor = 1. - strength * np.where(zenith < thresh_zen, 0, (zenith - thresh_zen) / (maxzen - thresh_zen))
         # For low zenith factor can be greater than one, so we need to clip it into a sensible range.
-        factor = clip(factor, 0, 1)
+        factor = np.clip(factor, 0, 1)
         return rayref * factor
 
 
-def get_reflectance_lut_from_file(filename):
+def _get_zeroed_result(shape, compute, chunks=None):
```

Comment:
could `np.zeroes_like` be used here (or the dask counterpart)?

Reply:
Ok I think this is done.